plugins/commands/migrate/src/main/kotlin/MigrateCommand.kt

```diff
@@ -169,3 +182,12 @@ private fun convertHoconToYaml(hocon: String): String {
 
     return yamlLines.joinToString("\n")
 }
+
+/**
+ * Return the path for the directory to store [PackageConfiguration]s for this [Identifier].
+ *
+ * TODO: Replace with equivalent function from the package configuration provider for the ort-config repository, once
+ *       implemented.
+ */
+fun Identifier.toPackageConfigurationDir() =
+    "${type.encodeOr("_")}/${namespace.encodeOr("_")}/${name.encodeOr("_")}/${version.encodeOr("_")}"
```

Comment:
Can be shortened to `toPath(emptyValue = "_")` like done in the [*helper-cli*'s `CreateCommand` for package configurations](https://github.com/oss-review-toolkit/ort/blob/5a92da88ce2176de3bb9e979a39a3e54cecb2282/helper-cli/src/main/kotlin/commands/packageconfig/CreateCommand.kt#L152C25-L152C50).